mesh/pncmesh.cpp

```diff
@@ -972,8 +985,8 @@ void ParNCMesh::GetFaceNeighbors(ParMesh &pmesh)
    std::map<int, int> vert_map;
    for (int i = 0; i < fnbr.Size(); i++)
    {
-      Element* elem = fnbr[i];
-      mfem::Element* fne = NewMeshElement(elem->geom);
+      auto* elem = fnbr[i];
```

Comment:
I suggest reverting this change is it hurts readability. Especially here, where it's good to know explicitly that `elem` is the internal NCMesh element and `fne` is the regular MFEM element.